build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
@gep13 well OpenCover  takes an `Action<ICakeContext> test` if we defined it before OpenCover is called it could be a simple `if` statement if it should be passed to open cover or directly evaluated against context as `test(Context);`.

just a thought :wink:  and 32-bit is just a hunch, could be that Azure Websites are more limited in allowing the kind of dll interop open cover is doing.